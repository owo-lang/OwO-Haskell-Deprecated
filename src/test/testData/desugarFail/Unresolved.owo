module Unresolved where

 a = b
