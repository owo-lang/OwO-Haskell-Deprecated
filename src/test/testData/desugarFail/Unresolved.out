File type: owo
Top module name: .Unresolved
UnresolvedReferenceError (Name ((30 3 6 in Just "testData/desugarFail/Unresolved.owo"), (31 3 7 in Just "testData/desugarFail/Unresolved.owo")) "b")
